kfp/kfp_support_lib/kfp_v1_workflow_support/Makefile

```diff
@@ -28,35 +29,16 @@ set-versions:
 	cat pyproject.toml | sed -e 's/"kfp\([=><][=><]\).*",/"kfp\1$(KFP_v1)",/' > tt.toml
 	mv tt.toml pyproject.toml
 
-build:: set-versions venv
-ifeq ($(KFPv2), 1)
-	# we want to prevent execution of the rule, when we run `make build` in upper directories and KFPv2==1
-	echo "Skipping build as KFPv2 is defined"
-else
-	@# Help: Build the distribution for publishing to a pypi
-	$(MAKE) build-dist
-endif
-
-build-dist :: set-versions .defaults.build-dist
+build:: .check-env .defaults.build-dist
```

Comment:
Please change `.check-venv` to `set-versions` so that `build` and `publish` targets will be similar to the ones in https://github.com/IBM/data-prep-kit/blob/dev/data-processing-lib/ray/Makefile